.travis.yml

```diff
@@ -16,7 +16,7 @@ addons:
       - g++-6
 
 install:
-    - g++ -v
+    - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-6 90
```

Comment:
whats the 90 for :3?

Reply:
don't know actually, copy pasted from somewhere